### ISSUE TITLE
axes tick_params label color not respected when showing scientific notation for axes scale

### ISSUE THREAD

**Issue by mgrady3** [matplotlib/matplotlib#7228]
Matplotlib version 1.5.1 (OS X.11.4 and Ubuntu 16.04) installed via pip

When you manually set the color for the axis ticks via
    
    ax.tick_params(labelcolor='w')

Then if the scale of the plot changes to the point where mpl decided to write the scale in scientific notation above the axis, the color of the scientific notation text does not obey the user defined color

There may be a reason for this but I don't see why it wouldn't follow the user-defined labelcolor for the axis ticks.

I'm not sure of providing a minimum working example. I would guess just creating an mpl figure without setting specific axes limits, manually setting the tick colors to a specific color, then plotting data that is some many orders of magnitude higher than the current axes scale should reproduce the error/bug/feature. The scientific notation text for the axis scale will appear in default black even if you have set the axis tick_params labelcolor to white.

An example is shown in the attached picture: The y-axis of the plot has a large enough scale to warrant using scientific notation. However the scientific notation label appears in black when the rest of the plot has the labelcolor set to white.

![axes-labelcolor-bug](https://cloud.githubusercontent.com/assets/7071404/19198537/b7b367da-8c8d-11e6-9054-9c9d2a081301.png)

**Comment by tacaswell** [comment]
Can you confirm that this is still broken in 2.0.0b4 `pip install --pre matplotlib`.

**Comment by mgrady3** [comment]
This is indeed fixed in 2.0.0b4

**Comment by NelleV** [comment]
I am not able to reproduce with the following code:

```python
import numpy as np
import matplotlib.pyplot as plt

X = np.arange(1, 16, 1) * 1e6
fig, ax = plt.subplots(facecolor="#000000")
ax.plot(X, X)
ax.tick_params(axis="x", colors="red")
```

Do you mind writing a small code snippet reproducing the bug?

**Comment by efiring** [comment]
Given that it is fixed in 2.0, I am closing this.
